code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -109,12 +115,59 @@
 		// we don't need to manage that.
 		new_tail.Insert(C, TRUE, FALSE)
 
+/datum/species/lizard/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
+	C.mob_surgery_speed_mod += 0.35
+	UnregisterSignal(C, COMSIG_CARBON_GAIN_ORGAN)
+	UnregisterSignal(C, COMSIG_CARBON_LOSE_ORGAN)
+	C.remove_client_colour(/datum/client_colour/monochrome/lizard)
+	return ..()
+
 /datum/species/lizard/randomize_main_appearance_element(mob/living/carbon/human/human_mob)
 	var/tail = pick(GLOB.tails_list_lizard)
 	human_mob.dna.features["tail_lizard"] = tail
 	mutant_bodyparts["tail_lizard"] = tail
 	human_mob.update_body()
 
+/datum/species/lizard/on_tail_lost(mob/living/carbon/human/tail_owner, obj/item/organ/tail/lost_tail, on_species_init)
+	. = ..()
+	if(!.)
+		return
+	if(lost_tail.type in mutant_organs)
+		RegisterSignal(tail_owner, COMSIG_MOVABLE_MOVED, .proc/on_move)
+
+/datum/species/lizard/on_tail_regain(mob/living/carbon/human/tail_owner, obj/item/organ/tail/found_tail, on_species_init)
+	. = ..()
+	if(!.)
+		return
+	if(found_tail.type in mutant_organs)
+		UnregisterSignal(tail_owner, COMSIG_MOVABLE_MOVED)
+
+/datum/species/lizard/clear_tail_moodlets(mob/living/carbon/human/former_tail_owner)
+	. = ..()
+	UnregisterSignal(former_tail_owner, COMSIG_MOVABLE_MOVED)
+
+/datum/species/lizard/proc/on_gained_organ(mob/living/receiver, obj/item/organ/tongue/organ)
+	SIGNAL_HANDLER
+
+	if(!istype(organ) || organ.taste_sensitivity > LIZARD_TASTE_SENSITIVITY)
+		return
+	receiver.remove_client_colour(/datum/client_colour/monochrome/lizard)
+
+/datum/species/lizard/proc/on_removed_organ(mob/living/unceiver, obj/item/organ/tongue/organ)
+	SIGNAL_HANDLER
+
+	if(!istype(organ) || organ.taste_sensitivity > LIZARD_TASTE_SENSITIVITY)
+		return
+	unceiver.add_client_colour(/datum/client_colour/monochrome/lizard)
+
+/datum/species/lizard/proc/on_move(mob/living/mover, atom/old_loc, movement_dir, forced, list/old_locs)
+	SIGNAL_HANDLER
+
+	if(!movement_dir || !prob(1))
+		return
+	mover.Knockdown(1 SECONDS)
+	to_chat(mover, span_warning("You trip from your imbalance!"))
```

Comment:
the severe negative moodlet and the inability to *wag (the most important feature of playing a lizard) are already enough of a punishment, IMO; tailless lizards don't need to be given brawl tripping too.

Reply:
discuss it with eob dont code review it